packages/account/src/Components/currency-selector/radio-button.tsx

```diff
@@ -1,11 +1,24 @@
-import React from 'react';
+import React, { InputHTMLAttributes, AllHTMLAttributes, ReactElement } from 'react';
 import classNames from 'classnames';
 import { Localize } from '@deriv/translations';
 import { Popover, Icon } from '@deriv/components';
 import { getCurrencyDisplayCode } from '@deriv/shared';
+// import { TReactElement } from 'Types';
 
-const USTPopover = ({ id }) => {
-    let popover_message;
+type TUSTPopover = {
+    id: string;
+};
+
+type TRadioButtonExtend = {
```

Comment:
We are using a different naming convention on deriv.com. I'm not sure we have this on deriv-app or not, but it is better to check it with TLs.
The way we are naming our types in deriv.com is like this: 
`component name + types || props`
example: `CtaBinaryProps` || `SectionTypes`